spring-rest/src/main/java/net/baeldung/ApplicationMain.java

```diff
@@ -0,0 +1,20 @@
+package net.baeldung;
```

Comment:
Normally, the convention is com, org, then net (used rarely).  I thought it easier to simply and cleanly add my code into a separate grouping. However, I think I'm supposed to move this over to Spring Angular instead. So... my point is moot.